lib/mapper.js

```diff
@@ -6,6 +6,47 @@ var traverse = require('isodate-traverse');
 var clone = require('component-clone');
 var unixTime = require('unix-time');
 var objCase = require('obj-case');
+var _ = require('underscore');
+var Track = require('segmentio-facade').Track;
+
+/**
+ * Turn a JSON product into Iterable format
+ * Specifically, any non-special fields should be nested under a field called "dataFields"
+ *
+ * @param products
+ * @returns {Array}
+ */
+function productsToIterableCart(products) {
+  return products.map(function(item){
+    var product = new Track({ properties: item });
+    // TODO - which of these fields do we have to proxy
+    var payload = {
+      id: product.id(),
+      sku: product.sku(),
+      name: product.name(),
+      categories: [product.category()],
+      price: product.price(),
+      quantity: product.quantity()
```

Comment:
all of these are safe and have defined methods :) https://github.com/segmentio/facade/blob/master/lib/track.js#L63-L76